src/pages/studyView/table/FixedHeaderTable.tsx

```diff
@@ -38,14 +34,12 @@ const RVSDTtoStrType = {
 
 @observer
 export default class FixedHeaderTable<T> extends React.Component<IFixedHeaderTableProps<T>, {}> {
-    private _store: LazyMobXTableStore<T>;
+    @observable private _store: LazyMobXTableStore<T>;
```

Comment:
Making the whole `_store` observable may not be the best idea, but this is a quick fix for the re-rendering issues and helps removing the unused properties.